Gemfile

```diff
@@ -80,7 +80,6 @@ gem 'i18n', '~> 1.8.11'
 group :development, :test do
   gem 'aruba'
   gem 'ci_reporter_rspec'
-  gem 'climate_control'
```

Comment:
Can we squash these changes down so that the gemfile changes don't manifest in this PR at all (added in an earlier commit and deleted here in the same PR).